src/js/models/menubar.js

```diff
@@ -219,7 +219,7 @@ define(function (require, exports, module) {
                 (document !== null) &&
                 !document.unsupported &&
                 (document.layers !== null) &&
-                (document.layers.selectedNormalized.size > 1),
+                (document.layers.selected.size > 1),
```

Comment:
why is the "layers-selected-2" rule (above) still hanging on with `selectedNormalized` ?